plugins/com.google.cloud.tools.eclipse.util/templates/appengine/pom.xml.standard.ftl

```diff
@@ -25,28 +25,57 @@
 
   <dependencies>
     <!-- Compile/runtime dependencies -->
+<#if servletVersion == "2.5">
     <dependency>
       <groupId>javax.servlet</groupId>
-<#if servletVersion == "2.5">
       <artifactId>servlet-api</artifactId>
       <version>2.5</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet.jsp</groupId>
+      <artifactId>jsp-api</artifactId>
+      <version>2.1</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet.jsp.jstl</groupId>
+      <artifactId>jstl-api</artifactId>
```

Comment:
It looks like this is purely API interfaces, like `servlet-api`. I see errors when trying to use JSTL.
![selection_005](https://user-images.githubusercontent.com/10523105/31401215-40449f70-adc0-11e7-9c20-08df9ddcad78.png)
Maybe we need to provide an implementation unlike `servlet-api`. But it seems like we anyway want to provide an implementation: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2465#issuecomment-335545290

Reply:
The exact error message is

Can not find the tag library descriptor for "http://java.sun.com/jsp/jstl/core" | index.jsp | JSP Problem

Reply:
To clarify, if we use `<groupId>jstl</groupId>` and `<artifactId>jstl</artifactId>` (as we currently do), the error goes away. (BTW, the group ID should be `javax.servlet` rather than `jstl`.)

Reply:
Aside: the [StackOverflow JSTL wiki](https://stackoverflow.com/tags/jstl/info) notes that 
> Note that there's also a `jstl:jstl` dependency, but it's exactly the same file, only with a wrong group ID. Furthermore, there's also a `javax.servlet.jsp.jstl:jstl` dependency, but it is empty.